test/functional/tool_wallet.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+# Copyright (c) 2018 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test bitcoint-wallet-tool.
+"""
+
+import os
+import subprocess
+import textwrap
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_equal
+
+class ToolWalletTest(BitcoinTestFramework):
+    def set_test_params(self):
+        self.num_nodes = 1
+
+    def process(self, *args):
+        binary = self.config["environment"]["BUILDDIR"] + '/src/bitcoin-wallet-tool' + self.config["environment"]["EXEEXT"]
+        input = None
```

Comment:
Remove this line. It redefines the built-in function `input`. Furthermore it is unused :-)